tools/git-version.ts

```diff
@@ -77,28 +81,34 @@ const commitAndTag = async (
     content: contentBase64,
     branch,
     message: commitMessage,
+    // @ts-expect-error Random typescript error: property sha is not available on type { ..., sha: string, ... }
```

Comment:
Can you please explain?

Reply:
The types just don't match for some reason. Didn't investigate again but did when I upgraded the dependency for integreat a while back and came to no conclusion. Feel free to try to make this work, would be nice if you'd find the problem.